packages/office-ui-fabric-react/src/components/Shimmer/examples/Shimmer.Application.Example.tsx

```diff
@@ -1,8 +1,18 @@
 import * as React from 'react';
-import { Async } from 'office-ui-fabric-react/lib/Utilities';
 import { createListItems, IExampleItem } from '@uifabric/example-data';
 import { IColumn, buildColumns, SelectionMode, Toggle } from 'office-ui-fabric-react/lib/index';
 import { ShimmeredDetailsList } from 'office-ui-fabric-react/lib/ShimmeredDetailsList';
+import { useSetInterval, useConst } from '@uifabric/react-hooks';
+
+export interface IShimmerApplicationExampleState {
```

Comment:
This can be internal now
```suggestion
interface IShimmerApplicationExampleState {
```